Wallet connect now deprecated

I was integrating wallet adapter as suggested here[here](https://solana.com/developers/guides/wallets/add-solana-wallet-adapter-to-nextjs) but i got npm  warning of the sdk_v1 being deprecated and updated to v2.

error log:
```
 npm install --save     @solana/wallet-adapter-base     @solana/wallet-adapter-react     @solana/wallet-adapter-react-ui     @solana/wallet-adapter-wallets 
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn deprecated @walletconnect/types@1.8.0: WalletConnect's v1 SDKs are now deprecated. Please upgrade to a v2 SDK. For details see: https://docs.walletconnect.com/
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm warn deprecated @walletconnect/mobile-registry@1.4.0: Deprecated in favor of dynamic registry available from: https://github.com/walletconnect/walletconnect-registry
npm warn deprecated uuidv4@6.2.13: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm warn deprecated @toruslabs/openlogin-utils@3.0.0: Not supported. Pls upgrade
npm warn deprecated @toruslabs/openlogin-jrpc@3.2.0: Not supported. Pls upgrade
npm warn deprecated @walletconnect/qrcode-modal@1.8.0: WalletConnect's v1 SDKs are now deprecated. Please upgrade to a v2 SDK. For details see: https://docs.walletconnect.com/
npm warn deprecated @solana/web3.js@2.0.0: @solana/web3.js version 2.0 is now @solana/kit! Remove @solana/web3.js@2 from your dependencies and replace it with @solana/kit. As needed, upgrade all of your @solana-program/* dependencies to the latest versions that use Kit.
npm warn deprecated ripple-lib@1.10.1: ripple-lib is deprecated. Please migrate to xrpl.js using this migration guide: https://xrpl.org/xrpljs2-migration-guide.html

added 1123 packages, and audited 1326 packages in 2m

99 packages are looking for funding
  run `npm fund` for details

3 moderate severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
```

the new document suggests using https://docs.reown.com/overview.

Can i edit the [page](https://solana.com/developers/guides/wallets/add-solana-wallet-adapter-to-nextjs) with required changes?

--- a/apps/docs/content/cookbook/wallets/connect-wallet-react.mdx
+++ b/apps/docs/content/cookbook/wallets/connect-wallet-react.mdx
@@ -65,9 +65,16 @@ If you are creating a new Nextjs project, install the following dependencies:
 $ npm install @solana/web3.js \
     @solana/wallet-adapter-base \
     @solana/wallet-adapter-react \
-    @solana/wallet-adapter-react-ui \
-    @solana/wallet-adapter-wallets
+    @solana/wallet-adapter-react-ui
 ```
+
+<Callout type="info">
+  This example uses Wallet Standard discovery with `wallets={[]}`, so it does
+  not need the `@solana/wallet-adapter-wallets` bundle. That bundle includes
+  legacy adapters and can install deprecated WalletConnect v1 dependencies. If
+  you need WalletConnect protocol support specifically, use Reown's [Solana
+  Adapter](https://docs.reown.com/advanced/providers/solana-adapter).
+</Callout>
 
 ### Create Solana Provider
 

